ui/plugin-system/src/components/CalculationSelector/CalculationSelector.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { FormControl, TextField, Autocomplete } from '@mui/material';
+import { CALCULATIONS_CONFIG, CalculationConfig, CalculationType } from '../../model/calculations';
+
```

Comment:
There are a bunch of common patterns with `UnitSelector` in these autocompletes and typing. If we hit a third case like this (I think we probably will), we should extract some of those patterns into a common component in `components` that these selectors can leverage.